### PR TITLE
Update input.py

### PR DESCRIPTION
**Pull request by SohomGhosh10**
Adding some comments

### CODE REVIEW

File: basics/input.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # print("Enter a number:")
 # x = input()
 # x = int(x)
-x = int(input("Enter a number:"))
+x = int(input("Enter a number:")) # taking input and typecasting to int
```
Comment (AvikAgarwala) [inline basics/input.py:4]: @SohomGhosh10 I've completed the comment

File: basics/input.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
-print(type(x))
-print(x + 2)
\ No newline at end of file
+print(type(x)) # int
```
Comment (AvikAgarwala) [inline basics/input.py:6]: @SohomGhosh10 I've modified comment

File: basics/input.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+print(x + 2)
```
Comment (AvikAgarwala) [inline basics/input.py:7]: @SohomGhosh10 I've removed comment

File: basics/input.py (hunk 2 above)
Comment (SohomGhosh10) [reply basics/input.py:6]: Okk thanks

File: basics/input.py (hunk 3 above)
Comment (SohomGhosh10) [reply basics/input.py:7]: Ok

**Review by AvikAgarwala — COMMENTED** [review]
@SohomGhosh10 I've made some changes to the changes you proposed.

**Pull request merged**